plugins/invisible-typing/src/Settings.tsx

```diff
@@ -16,7 +16,7 @@ export function Settings(): React.ReactElement {
       </SwitchItem>
       <SwitchItem
         {...util.useSetting(cfg, "invisible", true)}
-        note="When enabled, invisibility applies to all channels. When disabled, invisibility is channel based.">
+        note="Whether invisibility applies to all channels. (Serves as default for channel's state when invisiblity is channel based)">
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_clarification):** Clarification in settings note enhances user understanding of functionality.

```suggestion
note="Invisibility can be applied globally to all channels or individually per channel. When global invisibility is enabled, it overrides individual channel settings. When disabled, each channel's invisibility can be configured separately, using this global setting as the default.">
```